backend/src/main/java/com/greenfoxacademy/backend/models/User.java

```diff
@@ -22,6 +23,7 @@
 @NoArgsConstructor
 public abstract class User implements UserDetails {
 
+  @Getter
```

Comment:
Why we need a @Getter when we have @Data above?

Reply:
It is not needed, I will delete it.